docs/builtin-gadgets/traceloop.md

```diff
@@ -161,3 +161,44 @@ CPU PID        COMM             NAME                                       PARAM
 6   150829     ls               write                                      fd=1, buf=5355360 bin   dev   etc   home  pro… 158
 6   150829     ls               exit_group                                 error_code=0                                                                                  ...
 ```
+
+### Filter syscalls
+
+If you are only interested by specific syscalls, you can use the `--syscall-filters` flag:
+
+```bash
+$ kubectl gadget traceloop start --syscall-filters openat,write,execve
+$ kubectl create ns test
+$ kubectl run --restart=Never -n test -ti --image=busybox mypod -- sh -c 'RANDOM=output ; echo "3*7*2" | bc > /tmp/file-$RANDOM ; cat /tmp/file-$RANDOM'
```

Comment:
Use `--rm` so that you don't need to manually delete the pod:

```suggestion
$ kubectl run --restart=Never -n test --rm -ti --image=busybox mypod -- sh -c 'RANDOM=output ; echo "3*7*2" | bc > /tmp/file-$RANDOM ; cat /tmp/file-$RANDOM'
```